packages/flutter/lib/src/material/input.dart

```diff
@@ -18,7 +18,8 @@ export 'package:flutter/services.dart' show TextInputType;
 const Duration _kTransitionDuration = const Duration(milliseconds: 200);
 const Curve _kTransitionCurve = Curves.fastOutSlowIn;
 
-/// A simple text input field.
+/// A simple undecorated text input field. You probably want to use [Input]
+/// instead.
```

Comment:
Newline after the first sentence.

Rather than just directing people elsewhere in this style, we usually say something like "This widget is useful if baz. If you want foo, consider bar."